sdk/Cargo.toml

```diff
@@ -99,6 +99,7 @@ solana-frozen-abi-macro = { workspace = true, optional = true, features = [
 ] }
 solana-instruction = { workspace = true }
 solana-native-token = { workspace = true }
+solana-packet = { workspace = true, features = ["bincode", "serde"] }
```

Comment:
I'm not sure what the pattern has been in the other SDK bust-up PRs you've been working through, but is there any particular reason to isolate these two features? Can you even use `bincode` without `serde`?

Reply:
I guess I see now that the `bincode` feature pulls in the `serde` feature. So do we need to declare both here?

Reply:
Want to make the serde functionality available without bincode for those who want it